test/native_test/native_test_bindings.dart

```diff
@@ -3,244 +3,180 @@
 /// Generated by `package:ffigen`.
 import 'dart:ffi' as ffi;
 
-/// Holds the Dynamic library.
-ffi.DynamicLibrary _dylib;
-
-/// Initialises the Dynamic library.
-void init(ffi.DynamicLibrary dylib) {
-  _dylib = dylib;
-}
-
-double Function1Double(
-  double x,
-) {
-  return _Function1Double(
-    x,
-  );
-}
-
-final _dart_Function1Double _Function1Double =
-    _dylib.lookupFunction<_c_Function1Double, _dart_Function1Double>(
-        'Function1Double');
-
-typedef _c_Function1Double = ffi.Double Function(
-  ffi.Double x,
-);
-
-typedef _dart_Function1Double = double Function(
-  double x,
-);
-
-double Function1Float(
-  double x,
-) {
-  return _Function1Float(
-    x,
-  );
-}
-
-final _dart_Function1Float _Function1Float = _dylib
-    .lookupFunction<_c_Function1Float, _dart_Function1Float>('Function1Float');
-
-typedef _c_Function1Float = ffi.Float Function(
-  ffi.Float x,
-);
-
-typedef _dart_Function1Float = double Function(
-  double x,
-);
-
-int Function1Int16(
-  int x,
-) {
-  return _Function1Int16(
-    x,
-  );
-}
-
-final _dart_Function1Int16 _Function1Int16 = _dylib
-    .lookupFunction<_c_Function1Int16, _dart_Function1Int16>('Function1Int16');
-
-typedef _c_Function1Int16 = ffi.Int16 Function(
-  ffi.Int16 x,
-);
-
-typedef _dart_Function1Int16 = int Function(
-  int x,
-);
-
-int Function1Int32(
-  int x,
-) {
-  return _Function1Int32(
-    x,
-  );
-}
-
-final _dart_Function1Int32 _Function1Int32 = _dylib
-    .lookupFunction<_c_Function1Int32, _dart_Function1Int32>('Function1Int32');
-
-typedef _c_Function1Int32 = ffi.Int32 Function(
-  ffi.Int32 x,
-);
-
-typedef _dart_Function1Int32 = int Function(
-  int x,
-);
-
-int Function1Int64(
-  int x,
-) {
-  return _Function1Int64(
-    x,
-  );
-}
-
-final _dart_Function1Int64 _Function1Int64 = _dylib
-    .lookupFunction<_c_Function1Int64, _dart_Function1Int64>('Function1Int64');
-
-typedef _c_Function1Int64 = ffi.Int64 Function(
-  ffi.Int64 x,
-);
-
-typedef _dart_Function1Int64 = int Function(
-  int x,
-);
-
-int Function1Int8(
-  int x,
-) {
-  return _Function1Int8(
-    x,
-  );
-}
-
-final _dart_Function1Int8 _Function1Int8 = _dylib
-    .lookupFunction<_c_Function1Int8, _dart_Function1Int8>('Function1Int8');
-
-typedef _c_Function1Int8 = ffi.Int8 Function(
-  ffi.Int8 x,
-);
-
-typedef _dart_Function1Int8 = int Function(
-  int x,
-);
-
-int Function1IntPtr(
-  int x,
-) {
-  return _Function1IntPtr(
-    x,
-  );
-}
-
-final _dart_Function1IntPtr _Function1IntPtr =
-    _dylib.lookupFunction<_c_Function1IntPtr, _dart_Function1IntPtr>(
-        'Function1IntPtr');
-
-typedef _c_Function1IntPtr = ffi.IntPtr Function(
-  ffi.IntPtr x,
-);
+class NativeLibrary {
```

Comment:
I'm confused with the diff of this file, this looks related to a previous PR.

Didn't our tests fail because we did not generate the correct file?

Please ensure we catch this in the future.